src/main/java/it/cnr/isti/hpc/wikipedia/article/Link.java

```diff
@@ -100,7 +132,7 @@ public int hashCode() {
 		final int prime = 31;
 		int result = 1;
 		result = prime * result
-				+ ((description == null) ? 0 : description.hashCode());
+				+ ((anchor == null) ? 0 : anchor.hashCode());
```

Comment:
I am not aware of a `.isNull()` method in Java. I looked it up and didn't find. I am also not sure what is wrong with checking an object for nullity as `object == null`.